torch/_subclasses/fake_tensor.py

```diff
@@ -761,6 +761,7 @@ def __init__(
         allow_non_fake_inputs=False,
         shape_env=None,
         static_shapes=None,
+        _allow_unsafe_data_ptr_access=True,
```

Comment:
Can we avoid adding this ctor option if this is not something we want around permanently ? Maybe take it from a config from functorch ?

Reply:
Yup